src/app/controllers/admin-console.server.controller.js

```diff
@@ -577,6 +577,7 @@ exports.getExampleFormsUsingAggregateCollection = function (req, res) {
           meta: {
             action: 'getExampleFormsUsingAggregateCollection',
             ip: getRequestIp(req),
+            trace: getTrace(req),
             url: req.url,
```

Comment:
Out of scope for this PR, but can you help create an issue for an utility function that returns the relevant log info from a `req`? Seems like the ip, trace, url, header combo is repeated throughout the application :P

Reply:
Ok!